js/main.js

```diff
@@ -85,4 +102,23 @@ document.addEventListener('click', function (event) {
     return;
   }
   changeView(event.target.getAttribute('data-view'));
+  if (event.target.innerText === 'NEW') {
+    $newImgURL.setAttribute('src', 'images/placeholder-image-square.jpg');
+    $formEnterNew.reset();
+    $h1NewAndEditEntry.textContent = 'New Entry';
+  }
+});
+
+$ul.addEventListener('click', function (event) {
+  if (event.target.nodeName !== 'I') {
+    return;
+  }
+  changeView('entry-form');
+  $h1NewAndEditEntry.textContent = 'Edit Entry';
+  var $liEntryID = event.path[3].getAttribute('data-entry-id');
```

Comment:
Why `event.path[3]`? What are you getting here? Could you possibly use `closest` or something to that effect to simplify this and make it more readable?